Recrutify/Recrutify.DataAccess/Repositories/UserRepository.cs

```diff
@@ -23,8 +24,30 @@ public Task<User> GetByEmailAsync(string email)
 
         public Task<List<User>> GetByRolesAsync(List<Role> roles)
         {
-            var filter = _filterBuilder.AnyIn(u => u.Roles, roles);
-            return GetCollection().Find(filter).ToListAsync();
+            //var filter = _filterBuilder.ElemMatch(u => u.ProjectRoles,
```

Comment:
if this method doesn't work - get rid of it, and get rid of another useless code too